File: back-end/foody/src/main/java/com/seniorproject/foody/dto/InquiryInstruction.java
package com.seniorproject.foody.dto;

import com.seniorproject.foody.entities.Customer;
import com.seniorproject.foody.entities.Inquiry;
import com.seniorproject.foody.entities.InquiryDetail;
import lombok.Data;

import java.util.Set;

@Data
public class InquiryInstruction {
    // realationship customer,inquiry, inquirydetail[]
    private Customer customer;
    private Inquiry inquiry;
    private Set<InquiryDetail> inquiryDetails;
}

File: front-end/angular-foody-app/src/app/common/inquiry-instruction.ts
import { Customer } from "./customer";
import { Inquiry } from "./inquiry";
import { InquiryDetail } from "./inquiry-detail";

export class InquiryInstruction {
    customer:Customer;
    inquiry:Inquiry;
    inquiryDetails:InquiryDetail[];
    constructor(){
        this.inquiryDetails = [];
    }
}
